index.html

```diff
@@ -318,5 +318,14 @@ <h2 class="h2">I'm always interested in hearing about new projects, so if you'd
     <div class="fin"></div>
 </footer>
 <!-- END OF FORM -->
+
+
+<!-- POP CONTAINER -->
+<section class="pop-container" id="pop-container">
+  <section class="pop-up">
+    
+  </section>
+</section>
```

Comment:
- [ ] Kindly check the requirements on the image itself

Yours:
![image](https://user-images.githubusercontent.com/62705947/209007922-8cb3167f-f90c-4d74-9e9b-e70a1982a284.png)
Design:
![image](https://user-images.githubusercontent.com/62705947/209007975-3b183d1e-05e5-44ee-90c8-6cd70f4fd3b5.png)

Reply:
- [ ] Same requirements are for mobile as well. One extra thing for mobile is that the X for closing the button is not aligned correctly, kindly check the design for it 
![image](https://user-images.githubusercontent.com/62705947/209008190-b784dd50-5976-40b2-8eda-e20aea1de25e.png)

Reply:
Well resolved 👍🏽